### ISSUE TITLE
Kanban board - Setup

### ISSUE THREAD

**Issue by jmontanholi** [jmontanholi/meals-rails#12]
# Our group has only two members.

* ## Gabriel Rendon
* ## João Montanholi
![image](https://user-images.githubusercontent.com/73256642/142934848-19fedbe9-d965-4353-85b4-2f75ab0a5350.png)

# LINK TO PROJECT: https://github.com/jmontanholi/meals-rails/projects/1
![](https://media2.giphy.com/media/uWlpPGquhGZNFzY90z/giphy.gif)

**Comment by henatan99** [comment]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a link to the project's page. I think you should provide [this link](https://github.com/jmontanholi/meals-rails/projects/1) on this issue page. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by jmontanholi** [comment]
@henatan99 excuse me, but this is a kanban board review request. 
![image](https://user-images.githubusercontent.com/73256642/142934848-19fedbe9-d965-4353-85b4-2f75ab0a5350.png)

**Comment by henatan99** [comment]
> @henatan99 excuse me, but this is a kanban board review request. ![image](https://user-images.githubusercontent.com/73256642/142934848-19fedbe9-d965-4353-85b4-2f75ab0a5350.png)

Sure @jmontanholi , but you see the project requirement which says you should paste the GitHub project link to the issue description. I saw that you have added it now. So you can submit another code review request.

**Comment by jmontanholi** [comment]
@henatan99 Yes, I didn't see it, I am terribly sorry for the confusion I made.

**Comment by o-rashad** [comment]
Asslam Alaikum team @gabrielyea @jmontanholi   ,😊

Hi👋🏽

This milestone is complete. :heavy_check_mark: 

Good luck with the upcoming milestones :muscle:  

# Congratulations! 🎉 

Happy coding!👏

Feel free to leave any questions or comments in the issue thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
